endoreg_db/management/commands/create_model_meta_from_huggingface.py

```
@@ -0,0 +1,115 @@
+            type=str,
+            default="multilabel_classification_colonoscopy_default",
+            help="Name of the labelset",
+        )
+        parser.add_argument(
+            "--meta_version",
+            type=str,
+            default="1",
+            help="Version for the model meta",
+        )
+
+    def handle(self, *args, **options):
```

Ruff (0.14.2)

Unused method argument: `args`

(ARG002)